src/main/java/core/basesyntax/BinaryString.java

```diff
@@ -7,6 +7,19 @@
      * а возвращает String с представлением этого числа в двоичном виде.
      */
     public String toBinaryString(int value) {
-        return null;
+        StringBuilder value2 = new StringBuilder();
+        if (value == 0) {
+            value2.append(0);
+        } else {
+            while (value >= 1) {
+                if (value % 2 > 0) {
```

Comment:
if (value % 2 > 0) {
                    value2.append(1);
                } else {
                    value2.append(0);
                }
--> we can simplify -->
binaryString.append(value % 2)